test/Makefile

```diff
@@ -0,0 +1,20 @@
+GOPATH ?= ${GOPATH}
+BINDIR ?= ${GOPATH}/bin
+
+.PHONY: all update-vendor dep clean install test
+
+VERSION=$(shell git describe --tags --always --dirty)
+
+test:
+	go test -v github.com/kinvolk/ocicert/dist/...
```

Comment:
`kinvolk/ocicert` -> `opencontainers/distribution-spec`

Reply:
ah, got that one now as well.